setup.py

```diff
@@ -14,17 +14,18 @@
 import os
 from os.path import join, dirname, sep, exists, basename, isdir
 from os import walk, environ
+from distutils.command.build_ext import build_ext
 from distutils.version import LooseVersion
 from distutils.sysconfig import get_python_inc
 from collections import OrderedDict
-from time import sleep, time
+from time import time
```

Comment:
do we still need the `time` import?

Reply:
The `time` import is new, it was added an hour ago by someone else for epoch based timestamps on builds. I rebased to master only ten ish minutes ago.

Reply:
Relevant pull request here: https://github.com/kivy/kivy/pull/6008

Reply:
Oh yes my bad I missed that `time()` call, thanks!